.changeset/smart-turkeys-bathe.md

```diff
@@ -0,0 +1,30 @@
+---
+'@backstage/plugin-catalog-backend': minor
```

Comment:
Would just be a patch bump here right? Don't really need an entry for the new plugin either as that'll just result in a bump in the next release without any changes.

Reply:
@freben thinks not. It's minor bump because we add new feature to it - location analyzer.